flows/actions/base_test.go

```diff
@@ -140,16 +140,16 @@ func testActionType(t *testing.T, assetsJSON json.RawMessage, typeName string) {
 		// inject the action into a suitable node's actions in that flow
 		actionsPath := []string{"flows", fmt.Sprintf("[%d]", flowIndex), "nodes", "[0]", "actions"}
 		actionsJson := []byte(fmt.Sprintf("[%s]", string(tc.Action)))
-		assetsJSON = test.JSONReplace(assetsJSON, actionsPath, actionsJson)
+		testAssetsJSON := test.JSONReplace(assetsJSON, actionsPath, actionsJson)
```

Comment:
not declaring a new variable here meant that changes to the assets was leaking between tests